qiskit/quantum_info/operators/symplectic/clifford.py

```diff
@@ -37,7 +37,21 @@
 
 
 class Clifford(BaseOperator, AdjointMixin, Operation):
-    """An N-qubit unitary operator from the Clifford group.
+    r"""
+    An N-qubit unitary operator from the Clifford group.
+
+    An N-qubit Clifford operator takes Paulis to Paulis via conjugation
+    (up to a global phase). More precisely, the Clifford group :math:`\mathcal{C}_N`
+    is defined as
+
+     .. math::
+
+        \mathcal{C}_N = \{ U \in U(2^N) | U \mathcal{P}_N U^{\dagger} = \mathcal{P}_N \} / U(1)
+
```

Comment:
What does the ${}/ U(1)$ mean in this context?  I'm familiar with ${}\setminus U(1)$ to mean "subtract U(1)".  From the definition, I'm guessing it's meant to represent normalisation over global phases?

It's totally fine if it's standard notation - I'm probably just not familiar with it, since set theory is not my strong suit.

Reply:
the meaning of this symbol is taking the quotient group, and U(1) represents the scalar unitary matrices, so it generally means "up to a global phase". I'll add some further explanation.

Reply:
Ah cool, if it's standard notation that's all fine by me.  I'd guessed that it was _something_ to do with the phase-fixing (since $U(1)$ was clearly just the unitary scalars), but I don't always know the set-theory symbols.

Reply:
it's group theory and not set theory :)
as a mathematician one should be very precise :)